src/vcpkg/commands.integrate.cpp

```diff
@@ -467,7 +467,7 @@ namespace vcpkg::Commands::Integrate
                        .string_arg("-ExecutionPolicy")
                        .string_arg("Bypass")
                        .string_arg("-Command")
-                       .string_arg(fmt::format("& {& '{}' }", script_path));
+                       .string_arg(fmt::format("& {{& '{}' }}", script_path));
```

Comment:
Thank you for improving the identification of the `fmt` library. :)

For the modified content: I think it is to enable the fnt library to correctly identify a situation:

If `script_path` contains regular '{}', these braces will not be incorrectly recognized as placeholders in the formatted string.
eg: script_path: "/path/to/my{}script"
Before: `"& /path/to/myscript"`
After: `"/path/to/my{}script"`

Of course, using double curly braces also conforms to the escape rules of the `fmt` library.
But you mentioned that https://github.com/microsoft/vcpkg/issues/30283 can be fixed, but I don't see such a problem in the path provided. 
Can you explain it? :)